recipes/vcf-annotator/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+pip install PyVCF==0.6.8
```

Comment:
please depend on pyvcf instead.

Reply:
Thank you for the update. Should I submit a recipe for PyVCF to bioconda?

Reply:
Disregard that, just saw pyvcf was moved to conda-forge, will update